sdk_container/src/third_party/coreos-overlay/coreos-base/afterburn/files/0004-lto-thin.patch

```diff
@@ -1,29 +0,0 @@
-From 28a17874ace612fa1c93f6cfb46b9f5982027833 Mon Sep 17 00:00:00 2001
-Message-Id: <28a17874ace612fa1c93f6cfb46b9f5982027833.1686823250.git.dpark@linux.microsoft.com>
-From: Dongsu Park <dpark@linux.microsoft.com>
-Date: Thu, 15 Jun 2023 12:00:33 +0200
-Subject: [PATCH] Cargo: set LTO to false
-
-Set LTO to false (a.k.a. "thin-local") to avoid build failures
-related to LLVM, especially with Rust 1.70.0+.
-
----
- Cargo.toml | 2 +-
- 1 file changed, 1 insertion(+), 1 deletion(-)
-
-diff --git a/Cargo.toml b/Cargo.toml
-index e5b3dc30..46bbda69 100644
---- a/Cargo.toml
-+++ b/Cargo.toml
-@@ -22,7 +22,7 @@ name = "afterburn"
- path = "src/main.rs"
- 
- [profile.release]
--lto = true
-+lto = false
```

Comment:
To reduce the binary size we can set `codegen-units = 1` here, which is similar to thin LTO.

Full LTO would be ideal for binary size but since it's currently unclear if that works well with arm64, let's look at that later.

Reply:
`codegen-units = 1` looks like a good idea. It reduces binary size of amd64 at least by 50MiB.

Reply:
I think we also strip debug symbols but I need to find out where because the Cargo.toml doesn't do it. To get the same results directly we can set `strip = true` here, too.

Reply:
Done.